vanilla-framework/vanilla-docs-theme
# Update and import all Vanilla patterns

## Done
Updated the Vanilla-framework dependency. Included all aspects of the framework. Remove duplicates and integrated the framework variables.

## QA
- Pull the code and run `./run build`
- Open demo/index.html
- Check that everything looks fine
- Run `./run test` and check there are no errors
- Check hr styling works from Vanilla as its part of one of the issues

## Details
Fixes https://github.com/vanilla-framework/vanilla-docs-theme/issues/38 https://github.com/vanilla-framework/vanilla-docs-theme/issues/33
